packages/devtools_app/lib/src/screens/vm_developer/vm_class_screen.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../primitives/utils.dart';
+import 'vm_class_screen_controller.dart';
+import 'vm_developer_common_widgets.dart';
+
+/// A screen on the object inspector historyViewport displaying information related to class objects in the Dart VM.
+class VmClassScreen extends StatelessWidget {
+  const VmClassScreen({
+    required this.controller,
+  });
+
+  final ClassScreenController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: controller.refreshing,
+      builder: (context, refreshing, _) {
+        return Flexible(
+          child: Row(
+            children: [
+              Flexible(
+                child: Column(
+                  children: [
+                    Flexible(
+                      flex: 10,
```

Comment:
Why did we choose these flex values?

Reply:
Because the upper VMInfoCard was overflowing with the default flex values, will make it scrollable soon.